src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Wasi.props

```diff
@@ -0,0 +1,19 @@
+<!--
+***********************************************************************************************
+Microsoft.NETCore.Native.Wasi.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+This file defines Windows-specific steps in the build process for native AOT compilation.
+
+Licensed to the .NET Foundation under one or more agreements.
+The .NET Foundation licenses this file to you under the MIT license.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Part of workaround for lack of secondary build artifact import - https://github.com/Microsoft/msbuild/issues/2807 -->
+  <!-- required Target, but nothing to do for Wasm -->
+  <Target Name="SetupOSSpecificProps" />
+</Project>
```

Comment:
I think we should rather rename the `.Browser` one to `.wasm` and use it for both. We should eventually migrate the various knobs to this file from `.Native.targets`.

Reply:
Done, thanks